lib/spack/docs/chain.rst

```diff
@@ -21,8 +22,8 @@ you can add it as an entry to ``upstreams.yaml``:
     spack-instance-2:
       install_tree: /path/to/another/spack/opt/spack
 
-``install_tree`` must point to the ``opt/spack`` directory inside of the
-Spack base directory.
+``install_tree`` must point to the ``.../spack/opt/spack`` directory inside of 
+the Spack base directory.
```

Comment:
This is incorrect, since there is no `.../spack/opt/spack` directory inside of the Spack base directory. The original is clearer; not every install of spack lives in a directory `spack`. Many live in e.g. `v0.22.0` to allow multiple versions to co-exists.

The documentation can be improved, however, by making sure it clear that you can point to any `install_tree`, not just the default location in `opt/spack`.


How about this?
```suggestion
``install_tree`` must point to the ``opt/spack`` directory inside of the
 Spack base directory, or the location of the ``install_tree`` defined
 in :ref:`config.yaml <config-yaml>`.
```

Reply:
I do like it, it is clear. `/opt` is general enogh

Reply:
If you agree, please feel free to commit these changes (and address the other comment). Then I can review/approve and set this to merge.